assets/sass/components/settings/_googlesitekit-settings-module.scss

```diff
@@ -127,6 +127,16 @@
 		margin: 0 0 8px 0;
 	}
 
+	.googlesitekit-settings-module__fields-group {
+		border-bottom: 1px solid $c-mercury;
```

Comment:
Maybe an added `<hr>` would be more appropriate? I think it might also be a good idea to apply the same max-width (1098) to it as we do for the GA4 notice to keep it from spanning the entire area. What do you think?

Reply:
I considered it at the beginning but it didn't work well.